Description: Additional changes to make Botan2 work with QtCreator
Author: Adam Majer <amajer@zombino.com>
Origin: other
Last-Update: 2018-03-17
--- a/src/libs/3rdparty/botan/botan.pri
+++ b/src/libs/3rdparty/botan/botan.pri
@@ -4,7 +4,7 @@ HEADERS += $$PWD/botan.h
 equals(USE_SYSTEM_BOTAN, 1) {
     DEFINES += USE_SYSTEM_BOTAN
     CONFIG += link_pkgconfig
-    PKGCONFIG += botan-1.10
+    PKGCONFIG += botan-2
 } else {
 
 SOURCES += $$PWD/botan.cpp
